src/synth/dssi_op.ml

```diff
@@ -125,6 +125,11 @@ class dssi ~kind ?chan plugin descr outputs params source =
         done
   end
 
+let () =
+  Lang.add_module "dssi";
+  Lang.add_module "synth.dssi";
```

Comment:
Both?

Reply:
For this one we need both: we currently have all the plugins registered as `synth.dssi.*` and we have the function `dssi.register` to manually register a plugin.